data/mods/blazemod/vortex.json

```diff
@@ -0,0 +1,336 @@
+[
```

Comment:
Messy, these were. Something to do with automated versions to redact, and I had to change the turret fields to reflect vanilla: but generally- unchanged. Could use a template, not sure how popular vortex stuff is amongst players though.